src/plasoscaffolder/frontend/controller/sqlite_controller.py

```diff
@@ -51,18 +60,21 @@ def SourcePath(self, ctx: click.core.Context, param: click.core.Option,
     """
     while not self._plugin_helper.FolderExists(value):
       value = self._output_handler.PromptError(
-          'Folder does not exists. Enter correct one: ')
+          'Folder does not exists. Enter correct one')
     self._path = value
```

Comment:
sorry my bad, not paying close enough attention, missed it in the UI that value is a method argument and already set.